### PR TITLE
Add upper nav bar (not view selection) for document view

### PR DESCRIPTION
**Pull request by wendi-yu**
Adds the upper nav bar, with the document name and comment and share buttons. I stubbed out the user profile stuff in a separate component, since I'm assuming we'll eventually want to add in actual complex functionality once we have auth and so on implemented.

Also did some miscellaneous css cleanup

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
       <Router>
-      <Navbar />
         <Routes>
           <Route path='/' exact element={<Home/>} />
-          <Route path='/Document' element={<Document/>} />
+          <Route path='/document' element={<Document/>} />
         </Routes>
       </Router>
-    </>
+    </div>
```
Comment (RichardYSun) [inline src/App.jsx:15]: Thanks for the cleanup

File: src/components/document/Header/UserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import ProfileIcon from "@/assets/pfp.svg"
+
+const UserProfile = () => {
+    // This is a fill in component.
+    // TODO: fill out this functionality (with contexts, whatever) once we have login/authentication
```
Comment (RichardYSun) [inline src/components/document/Header/UserProfile.jsx:5]: is there a way to link up Linear so it adds TODOs auto? (I've added link to this comment in linear task)

**Review by RichardYSun — APPROVED** [review]
lgtm

File: src/components/document/Header/UserProfile.jsx (hunk 2 above)
Comment (wendi-yu) [reply src/components/document/Header/UserProfile.jsx:5]: I have no idea actuallyl lol not that I know of

**Pull request merged**